miio/integrations/petfooddispenser/status.py

```diff
@@ -0,0 +1,67 @@
+import enum
+from typing import Any, Dict
+
+from miio import DeviceStatus
+
+class FoodStatus(enum.Enum):
+    Normal = 0
+    Low = 1
+    Empty = 2
+
+class PowerState(enum.Enum):
+    Mains = 0
+    Battery = 1
+
+class PetFoodDispenserStatus(DeviceStatus):
+    """Container for status reports from the Pet Feeder / Smart Pet Food Dispenser."""
+
+    def __init__(self, data: Dict[str, Any]) -> None:
+        """
+        Response from pet feeder (mmgg.feeder.petfeeder):
+ 
+        {'food_status': 0, 'feed_plan': 1, 'door_status': 0,
+          'feed_today': 0, 'clean_time': 7, 'power_status': 0,
+          'dryer_days': 6, 'food_portion': 0, 'wifi_led': 1,
+          'key_lock': 1, 'country_code': 255,}
+        """
+        self.data = data
+
+    @property
+    def food_status(self) -> str:
+        """Current food status / level."""
+        return FoodStatus(self.data["food_status"])
+
+    @property
+    def feed_plan(self) -> bool:
+        """Automatic feeding status."""
+        return bool(self.data["feed_plan"])
+
+    @property
+    def door_status(self) -> bool:
+        """Food bin door status."""
+        return bool(self.data["door_status"])
+
+    @property
+    def clean_days(self) -> int:
```

Comment:
Use `timedelta` for this and the dryer one, this makes it consistent with other integrations.
Also, use naming scheme `<variable>_left`.

Reply:
Perfect, was just about to ask about this.
"dryer_days" > "desiccant_left" 

Can't quite think how to phrase "clean_days". 
It's the number of days until the unit requires cleaning.
Suggestions?

Reply:
I don't have a good term for that either, `time_until_clean_left` is sounds quite verbose.

Reply:
Yeah that's what I was trying to avoid, even though it's very pythonic to make it as long as is needed to convey it's use.